taskcluster/ci/tests/kind.yml

```diff
@@ -85,6 +90,12 @@ tasks:
           max-run-time: 3600
       description: "Test the full `translations_taskgraph` to validate the latest changes"
       run-on-tasks-for: ["github-push", "github-pull-request"]
+      optimization:
+        skip-unless-changed:
+            - pipeline/**
```

Comment:
@eu9ene Please double check that I'm covering the required folders here and above.